### DISCUSSION TITLE
GSoC 2021

### DISCUSSION THREAD

**Discussion by avivijay19** [google/android-fhir#298] [General]
Hello,
Myself Avinash Vijayvargiya, A Second Year student form VIT,Pune
https://github.com/avinash14022002

Wanted to contribute to this project can someone guide me

**Comment by VarunT11** [comment]
Hello Everyone!
I am Varun Tiwari, a 2nd Year Undergraduate hailing from India and pursuing B.Tech in Information Technology from Indian Institute of Information Technology, Allahabad. I have an experience of more than a year in Android App Development (Java) and a little bit Web Development (NodeJS, PHP). I have been contributing to the Open Source for quite a time. I along with other open-source enthusiasts of my Institute have developed an Open-Source Application which could help Universities to track their Daily Events.
I would be looking for opportunities to contribute to the Google FHIR SDK and move ahead in my journey with Open Source. I would be applying as a student for the Google Summer of Code – 2021.
You can check my Social Presence at the following:
GitHub : https://github.com/VarunT11
LinkedIn : https://www.linkedin.com/in/varun-tiwari-0251921b6/
Looking forward for a great experience ahead!
Thank You

**Comment by Manikant25** [comment]
Hello Everyone👋
I am Manikant Rai, a 2nd-year computer science undergraduate from SRM institute of Science And Technology.  I have 3 months of extensive experience with android which I have come up with building small projects of every concept basically **jetpack components** like fragments, navigation, ViewModel-Live data, etc.  I know how important is architecture while developing an app for scalability and efficiency. I am familiar with **MVVM** (Model–View–ViewModel)  architecture.  I have contributed to open source in past and looking to contribute more. 
I would like to work on android-fhir projects and participate in GSoC 2021. 
Here is my GitHub and LinkedIn Handle
`Github` -https://github.com/Manikant25
`LinkedIn` - https://www.linkedin.com/in/manikant-rai-544179136/

Thank you

**Comment by jingtang10** [comment]
Hi @avinash14022002 @VarunT11 and @Manikant25!

Thank you for your interest.

As an open source codebase we always welcome contribution, so please do take a look at the open [issues](https://github.com/google/android-fhir/issues).

For GSoc applications, please use the official site as a guide for the [timeline](https://summerofcode.withgoogle.com/how-it-works/#timeline). I believe the applications will open on 29 March.

Thanks!

**Reply by avivijay19** [reply]
@jingtang10 sir,
Please let me know how to get started?

**Comment by prakharagarwal-dev** [comment]
Hi everyone!

I am Prakhar Agarwal, a 1st-year student at the Indian Institute of Technology, Varanasi a.k.a. IIT (BHU), pursuing Bachelor of Technology.  
I have been developing Android Apps from the past 2 years and since last 5 months I am also contributing to open-source organisation Amahi's Android app and I have fallen in love with open source ever since.

GSoC 2021 will be a great learning opportunity for me as I can get a chance to be mentored by world-class engineers from around the world.

I am proficient in Kotlin, Java, Android SDK, Android Jetpack, MVVM architecture including Modern Android Development and I am also familiar with FHIR API.

I am interested to work in ["Android FHIR Client"](https://github.com/google/android-fhir/wiki/GSoC-Project-Ideas#android-fhir-client) project as mentioned in the IdeaList for GSoC 2021. I would like to further discuss the project with you and I would love to be involved for the long-term.

`Amahi Android App repo` https://github.com/amahi/android
`List of contributions in Amahi` https://bit.ly/ListOfContrib
`GitHub` https://github.com/Prakhar-Agarwal-byte
`LinkedIn` www.linkedin.com/in/prakhar-agarwal-byte

**Comment by Abhinavtdk** [comment]
Hello all, my name is Sai Abhinav. I'm a 2nd year B-Tech student pursuing a degree in Computer Science. I've been learning android development the past few months and I've developed a few small apps along the road. I can use both Java and Kotlin to code, started with flutter recently.

I'm interested in contributing to your organization and since this is my first time applying for gsoc, it would be a great help if the mentors could guide me on the necessary things to do.

**Comment by harshhks2** [comment]
Hi Everyone. I am Harsh, a third year undergraduate student from IIT Kharagpur. I am very excited to be a part of this community. I want to start making contributions to the organization and continue it for GSOC 2021. It would be really helpful to know how and where to start.

**Comment by Syed-Affan-Hameed** [comment]
Hi Mentors and my fellow coders. I am Syed , I'm a 3rd year B.E Computer Science and Engineering student from JSS Science and Technology University(Formerly SJCE). I would be so glad and delighted to contribute to FHIR android as I think that technology and healthcare should go in synergy. I would like to participate in GSoC 2021 with this organisation. I am a trained android developer, proficient in kotlin. It is my first time applying to GSoC it would be nice if the mentors could get in touch with all of us interested students in this page to discuss the proposals beforehand in order to submit successful proposals.
Thanks!.

**Comment by rahulshah456** [comment]
Hello Everyone, 
I am Rahul Kumar Shah currently, a senior-year undergraduate student at GGSIPU, New Delhi, India pursuing my Bachelor's in Information Technology. I love working on new development projects which solve real-world problems.
I have been working on android for quite some time now and have collaborated with talented people to create digital products for both business and product use.
I have previously contributed to AnitaB.org for the development of mentorship application. And have published some android apps in the google play store with the latest architecture components.
I am interested to work on the android FHIR Client project and would love to be a part of the community.

But, have some queries regarding the project like MVP features that should be implemented before going into production and I am unable to find any stable communication channels or mailing lists to communicate with the mentors for further discussions.

Thanks

**Comment by kimr25302** [comment]
Hello Everyone! I am Kim D. Huynh, a Graduate student at Georgia Tech student studying Computer Science from Atlanta, Georgia. Previously, I have worked extensively on Android projects at Georgia Tech. Additionally, I have experience with contributing to software in the health field, specifically when I was a Software Engineering Intern at a regional hospital. 

In general, I enjoy learning about Mobile Software Development and relating the different ways to create a tangible impact on health. I first learned about Android FHIR through GSoC 2021, as it will be my first year applying.  I am really excited to begin contributing to Android FHIR SDK and help make interoperability in healthcare be more accessible.

Sincerely,
Kim D. Huynh

**Comment by deepankarb** [comment]
Hi all,

Please refer to Jing's comment above.

To get started, just clone the project and start playing around. Once you are familiar with the code you can try fixing an issue and submitting a PR. 

Tip: Look for "good first issue" tag when picking an issue to work upon.

Locking this conversation now.

Cheers.
